app/assets/stylesheets/fae/modules/forms/_colorpicker.scss

```diff
@@ -0,0 +1,57 @@
+.js-color-picker {
```

Comment:
The purpose of `.js-` prefixed classes is [that they never interact with CSS](http://nicolasgallagher.com/about-html-semantics-front-end-architecture#javascript-specific-classes). This should be a regular class instead (i.e. `.input-color-picker` or simply `.color-picker`) and that class name should be included when you're defining `input_class` on `fae_color_picker`